res/values/arrays.xml

```diff
@@ -5,6 +5,7 @@
     <item>azerty</item>
     <item>qwerty</item>
     <item>qwerty_lv</item>
+    <item>qwerty_ru</item>
```

Comment:
I think this layout isn't named qwerty. Wikipedia calls it [JCUKEN](https://en.wikipedia.org/wiki/JCUKEN).

There's an other layout that would be attributed the name "russian qwerty" ([phonetic layout](https://en.wikipedia.org/wiki/Phonetic_keyboard_layout)) so keep the name free.